src/api/user.js

```diff
@@ -0,0 +1,20 @@
+import { USER_OK } from 'test/mocks/user';
+import { makeRequest, METHODS } from 'api/apiManager';
+
+export const getUser = (params) => {
+  if (params) {
+    console.info(`calling API /user/${params}`);
+  }
+  return makeRequest({
+    uri: '/users/',
+    method: METHODS.GET,
+    mockResponse: USER_OK.payload,
+  });
+};
+
+export const putUser = user => makeRequest({
+  uri: '/users/',
+  method: METHODS.PUT,
+  mockResponse: USER_OK,
```

Comment:
Here is ok, but the generated response will be `{ payload: { payload: { ... user data ...} } }`.

After refactoring USER_OK mock removing the payload layer, this will be correct